packages/js/src/installation-success.js

```diff
@@ -117,7 +118,12 @@ function InstallationSuccessPage() {
 							</a>
 						</div>
 					</div>
-					<a id="installation-success-skip-link" className="yst-bottom-12 yst-right-0 yst-mr-5 yst-self-end yst-text-base md:yst-absolute" href={ "/wp-admin/admin.php?page=wpseo_dashboard" }>
+					<a
+						id="installation-success-skip-link"
+						className="yst-bottom-12 yst-right-0 yst-mr-5 yst-self-end yst-text-base md:yst-absolute"
+						href={ "/wp-admin/admin.php?page=wpseo_dashboard" }
+						data-hiive-event-name="clicked_skip_button | installation successful screen"
```

Comment:
are we sure about the missing underscores?

Reply:
I guess so: if you look at the sheet, each event name of the form: `event_name | context` follows the pattern